### DISCUSSION TITLE
The approval of the pull request

### DISCUSSION THREAD

**Discussion by fbasatemur** [enzifiri/GitHub-Achievements#681] [Q&A] [answered]
Why haven't you approved the pull request yet, bro?

https://github.com/enzifiri/GitHub-Achievements/pull/680

**Comment by enzifiri** [comment, marked as answer]
> Why haven't you approved the pull request yet, bro?
> 
> #680

Thanks for the pr looks great!

**Comment by ricoglr** [comment]
Çekme isteğimi kabul eder misin?

**Comment by ghost** [comment]
Hi
